### PR TITLE
VIH-7201 update interpreter scenario

### PR DESCRIPTION
**Pull request by shravankris**
https://tools.hmcts.net/jira/browse/VIH-7201

- Updated interpreter scenario and refactored participant list assessment

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by robin-weller — APPROVED** [review]

File: VideoWeb/VideoWeb.AcceptanceTests/Steps/HearingRoomSteps.cs [hunk 1]
```diff
@@ -164,17 +172,25 @@ public void ThenTheJudgeCanSeeInterpreterAndInterpreteeOnParticipantList()
         {
             var interpreter = _c.Test.ConferenceParticipants.FirstOrDefault(x => x.HearingRole.ToLower() == "interpreter");
             var interpretee = _c.Test.ConferenceParticipants.FirstOrDefault(x => x.Id == interpreter.LinkedParticipants.FirstOrDefault().LinkedId);
-
-            _browsers[_c.CurrentUser].Driver.WaitUntilVisible(HearingRoomPage.InterPreterName(interpreter.DisplayName)).Text.Should().Contain($"{interpretee.DisplayName}");
+            
+            var interpreterText = _browsers[_c.CurrentUser].Driver
+                                    .WaitUntilElementExists(HearingRoomPage.ParticipantPanel)
+                                    .FindElement(HearingRoomPage.InterPreterName(interpreter.DisplayName)).Text.Trim();
```
Comment (npoxon) [inline VideoWeb/VideoWeb.AcceptanceTests/Steps/HearingRoomSteps.cs:178]: I think you can get rid if FindElement as the WaitUntil element will find it

File: VideoWeb/VideoWeb.AcceptanceTests/Steps/HearingRoomSteps.cs [hunk 2]
```diff
@@ -121,6 +121,14 @@ public void ThenParticipantsCanSeeTheOtherParticipants(string user)
...
         }
 
+        [Then(@"the (.*) clicks raise hand")]
+        public void ThenTheParticipantClicksRaiseHand(string user)
+        {
+            _browserSteps.GivenInTheUsersBrowser(user);
+            _browsers[_c.CurrentUser].Driver.WaitUntilElementClickable(HearingRoomPage.ToggleHandRaised).Click();
+        }
+
+
```
Comment (npoxon) [inline VideoWeb/VideoWeb.AcceptanceTests/Steps/HearingRoomSteps.cs:131]: Extra line break

File: VideoWeb/VideoWeb.AcceptanceTests/Steps/HearingRoomSteps.cs [hunk 3]
```diff
@@ -164,17 +172,25 @@ public void ThenTheJudgeCanSeeInterpreterAndInterpreteeOnParticipantList()
...
+        public void ThenTheJudgeCanSeeInterpreterHandRaised(string status)
         {
+            if (status == "raised")
+                _browsers[_c.CurrentUser].Driver.WaitUntilVisible(HearingRoomPage.HandRaised).Displayed.Should().BeTrue();
+            else
+                _browsers[_c.CurrentUser].Driver.WaitUntilElementNotVisible(HearingRoomPage.HandRaised).Should().BeTrue();
         }
 
 
+
```
Comment (npoxon) [inline VideoWeb/VideoWeb.AcceptanceTests/Steps/HearingRoomSteps.cs:193]: Extra line break

**Review by npoxon — CHANGES_REQUESTED** [review]

File: VideoWeb/VideoWeb.AcceptanceTests/Steps/HearingRoomSteps.cs (hunk 1 above)
Comment (shravankris) [reply VideoWeb/VideoWeb.AcceptanceTests/Steps/HearingRoomSteps.cs:178]: This is finding element within a parent, first WaitUntil is to find participant panel and second is to find interpreter div within participant panel.

**Review by npoxon — APPROVED** [review]

**Pull request merged**